client/wfnews-war/src/main/angular/src/app/components/report-of-fire/compass-page/rof-compass-page.component.ts

```diff
@@ -18,14 +18,14 @@ interface DeviceOrientationEventiOS extends DeviceOrientationEvent {
 export class RoFCompassPage extends RoFPage implements OnInit {
   public compassFaceUrl: string
   public compassHandUrl: string
-  public compassHeading: number;
-  public currentLat: number
-  public currentLong: number
+  public compassHeading: number = 0;
```

Comment:
No need to change it, but just as an fyi in typescript if you assign a value like zero, you don't need to define the type as `number`

`public compassHeading = 0;` will auto-derive the type.

Reply:
Ok good to know